.travis.yml

```diff
@@ -43,7 +43,8 @@ install:
 
 script:
    - cd test
-   - coverage run --source=jpegenc -m py.test --include-reference
```

Comment:
@mkatsimpris no problem, I realize this is a work in progress, I wanted to make sure that these comments were not lost.